modules/manage/pages/kubernetes/kubernetes-rack-awareness.adoc

```diff
@@ -9,13 +9,17 @@ When you enable rack awareness in the Redpanda Helm chart, Kubernetes failure zo
 
 == Prerequisites
 
-Make sure that each of your Node resources is annotated with a key/value pair that corresponds to a failure zone. The Helm chart assigns each Redpanda broker to a particular rack, according to the failure zone of the Kubernetes node on which the broker is running.
+You must have the following:
 
-== Annotate Node resources
+* Kubernetes cluster: Ensure you have a running Kubernetes cluster, either locally, such as with minikube or kind, or remotely.
+
+* https://kubernetes.io/docs/tasks/tools/#kubectl[Kubectl^]: Ensure you have the `kubectl` command-line tool installed and configured to communicate with your cluster.
+
+== Annotate or label Node resources
 
 In Kubernetes, failure zones are treated as racks. Redpanda maps each rack to a failure zone and places partition replicas across them. For more details about Kubernetes failure zones, see the https://kubernetes.io/docs/setup/best-practices/multiple-zones/[Kubernetes documentation^].
 
-To assign a failure zone to your Kubernetes nodes, ensure that each of your Node resources is annotated with a key/value pair that corresponds to a failure zone. The Helm chart assigns each Redpanda broker to a particular rack, according to the failure zone of the Kubernetes node on which the broker is running.
+To assign a failure zone to your Kubernetes nodes, ensure that each of your Node resources is annotated or labeled with a key/value pair that corresponds to a failure zone. The Helm chart assigns each Redpanda broker to a particular rack, according to the failure zone of the Kubernetes node on which the broker is running.
 
 Managed Kubernetes platforms usually annotate Node resources with the availability zone in which the node instance is hosted. For example `topology.kubernetes.io/zone=us-east-1a`.
```

Comment:
I think usage of `us-east-1a` as AWS zone name is wrong example. AWS can change mapping of AWS zone ID to AWS zone name per AWS account.

> AWS maps the physical Availability Zones randomly to the Availability Zone names for each AWS account. This approach helps to distribute resources across the Availability Zones in an AWS Region, instead of resources likely being concentrated in Availability Zone "a" for each Region. As a result, the Availability Zone us-east-1a for your AWS account might not represent the same physical location as us-east-1a for a different AWS account. For more information, see [Regions and Availability Zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) in the Amazon EC2 User Guide.

REF:
https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html

If customer would like to do use rack awareness feature it would be better across AWS accounts, it would be better to stick to AWS Zone IDs.